moto/awslambda/models.py

```diff
@@ -983,6 +985,28 @@ def send_dynamodb_items(self, function_arn, items, source):
         func = self._lambdas.get_arn(function_arn)
         return func.invoke(json.dumps(event), {}, {})
 
+    def send_log_event(
+        self, function_arn, filter_name, log_group_name, log_stream_name, log_events
+    ):
+        data = {
+            "messageType": "DATA_MESSAGE",
+            "owner": "123456789012",
```

Comment:
Can  you use the ACCOUNT_ID from moto.core.models.py  here?

Reply:
Thanks, good point! Fixed and committed.